test/draw/RasterMonochromeTest.ooc

```diff
@@ -68,6 +68,11 @@ RasterMonochromeTest: class extends Fixture {
 			image2 = image2 resizeTo(size)
 			expect(image2 size == size)
 		})
+		this add("copy", func {
+			image := RasterMonochrome open(this sourceFlower)
+			image2 := image copy()
+			expect(image size == image2 size)
```

Comment:
Thanks everyone. `RasterMonochrome` was supposed to look for the super constructor
```ooc
init: func ~fromOriginal (original: This)
```
in `RasterPacked`, but with an argument of type `RasterImage`, it instead went looking further up in `RasterImage` where it found an appropriate constructor, which led to `this _stride` never being set.
Changing the constructor in `RasterPacked` to
```ooc
init: func ~fromOriginal (original: RasterImage)
```
passes all your tests.
Will push shortly.